src/popup/popup.css

```diff
@@ -20,15 +20,17 @@ button {
   font-size: 14px;
```

Comment:
The photon style guide also provide some guidance related to the size, spacing and font size of the common UI components, like buttons. Let's double-check those and try to match the suggestions, here is the page that describes the photon buttons styles: https://design.firefox.com/photon/components/buttons.html